modmesh/pilot/_base_app.py

```diff
@@ -322,6 +323,7 @@ class OneDimBaseApp(PilotFeature):
         - `plot_num` (bool): Flag indicating whether to plot numerical data.
         - `use_grid_layout` (bool): Flag indicating whether to use a grid
           layout.
+        - `adjust_region` (bool): Flag indicating whether to adjust the region.
```

Comment:
The current description sounds like the region will be automatically adjusted. However, based on the implementation, this flag seems to control whether the app allows the user to modify the number of regions (e.g., by showing Add/Delete buttons). Maybe we can consider rephrasing it to something like: "Flag indicating whether the user can modify the number of regions"